lib/src/draw/draw_line.dart

```diff
@@ -23,31 +24,56 @@ Image drawLine(Image image, int x1, int y1, int x2, int y2, int color,
   int dx = (x2 - x1);
   int dy = (y2 - y1);
 
+  var radius = (thickness / 2.0).floor();
+
   // Drawing a single point.
   if (dx == 0 && dy == 0) {
-    return drawPixel(image, x1, y1, color);
+    thickness == 1
+        ? drawPixel(image, x1, y1, color)
+        : fillCircle(image, x1, y1, radius, color);
+    return image;
   }
 
   // Axis-aligned lines
   if (dx == 0) {
     if (dy < 0) {
       for (int y = y2; y <= y1; ++y) {
-        drawPixel(image, x1, y, color);
+        if (thickness <= 1)

```

Comment:
Please use { } for all blocks, even single statement blocks.

Reply:
fixed